docs/source/technicaloverview.rst

```diff
@@ -4,100 +4,78 @@
+source database and INSERTing them into the target (copy) and applying the
+binlog changes from the source on the target (change synchronization).
+
+This setup has the advantage of working with essentially any configuration of
+MySQL. It also provides a seamless process as it is all contain in a single
+package, as opposed to split between multiple applications
+(mysqldump/xtrabackup and mysql replication).
+
+It is important to note that Ghostferry is simple a part of the puzzle of a
+live database migration. For data integrity reasons, Ghostferry mandates that
+you stop writes to the dataset you are copying a stage of execution known as
+cutover. During the same stage of execution, you'll also likely need to repoint
+your database from the source to the target.
+
+To gain a better understanding of the overall process, let's take a look how
+Ghostferry works:
 
 1. Ghostferry begins to pull binlog events from the source database, applying
    the entries that are applicable to the target. This continues in the
    background.
-2. Ghostferry SELECTs data from the source database and copies them to the
+2. Ghostferry SELECTs data from the source database and INSERTs them to the
    target database.
 3. Ghostferry finishes copying all data from the source and target database.
-   The binlog apply operation continues in the background.
-4. Wait until the binlog apply is roughly caught up to the latest available
-   position on the source database.
-5. Set the application to be read only for the data that Ghostferry has copied
-   (or even the whole source database, if possible). This is an out of band
-   step.
-6. Instruct Ghostferry to catch up to the latest available position on the
-   source database and stop synchronizing data.
-7. Change the application to use the target database and enable writes.
-
-The tool will cause some amount of downtime: between step 5 and step 7. The
-window of downtime is dependent on how fast those steps can be done and they
-should be relatively small (on the order of seconds - minutes).
+   The binlog apply operation of (1) continues in the background.
+4. Ghostferry waits until the binlog apply is roughly caught up to the latest
+   available position on the source database.
+5. Outside of Ghostferry: you should no more writes to to the source database
+   via either a read_only flag (for whole database) or some sort of application
+   level lock (for partial database).
+6. Instruct Ghostferry to enter the cutover phase. This essentially tells
+   Ghostferry to stop tailing binlog after catching up to the latest master
+   position. Once Ghostferry does this, it terminates. This step can be done
+   via the web ui or via a method call.
+7. Outside of Ghostferry: you should change the application that uses the
+   source database to use the target database. You should also enable writes on
+   the target database if it was previously read only somehow.
```

Comment:
`s/it was/they were/`?

Reply:
I think `it was` is correct here as I'm referring to a singular `target database` here.

Reply:
yea sorry, my bad, I misread this